cg/meta/workflow/rnafusion.py

```diff
@@ -134,26 +107,27 @@ def get_references_path(self, genomes_base: Optional[Path] = None) -> Path:
     def config_case(
```

Comment:
Would it be too much work to change this name to a function starting with a verb? It is easier to know what the function does just by reading the name. I am thinking of `set_config_case` but you may have better ideas

Reply:
same for tax profiler

Reply:
We are not setting anything, we are creating the directory and necessary files for the pipeline to run. This name is also used in balsamic, so it would make sense to change it for all pipelines. Let's leave it for now I would say so this PR doesn't become a monster